app/install.php

```diff
@@ -17,24 +17,10 @@
 	define('STEP', 0);
 }
 
-if (STEP === 3 && isset($_POST['type'])) {
+if (STEP === 2 && isset($_POST['type'])) {
```

Comment:
Uh… was it necessary to inverse these two steps in this PR? There are so many changes that I'm not sure of what changed exactly :/

Reply:
The reason for changing the order is that you cannot create the default user fully if the database is not in place first. It becomes similar than for the CLI, were one first create the DB and general config, and then create the default user and other users.

Reply:
In general, I am trying to remove the custom code in install.php and instead use the code we already have in the PDO classes, like for the CLI install.